pkg/core/config.go

```diff
@@ -47,6 +48,8 @@ type ClusterConfiguration struct {
 	Subcapacities        map[string][]string              `yaml:"subcapacities"`
 	Authoritative        bool                             `yaml:"authoritative"`
 	ConstraintConfigPath string                           `yaml:"constraints"`
+	OpenPolicyAgentPath  string                           `yaml:"open_agent_policy"`
+	PreparedRegoQuery    rego.PreparedEvalQuery           `yaml:"-"`
```

Comment:
I'm not anticipating much overlap in the checks that OPA policies will be used for on project level and domain level, respectively. Therefore please split domain-level and project-level checks into two separate policies:
```suggestion
	OPA  OPAConfiguration                           `yaml:"opa"`
```

where
```go
type OPAConfiguration struct {
  ProjectQuotaPolicyPath string `yaml:"project_quota_policy_path"`
  ProjectQuotaQuery rego.PreparedEvalQuery `yaml:"-"`
  DomainQuotaPolicyPath string `yaml:"domain_quota_policy_path"`
  DomainQuotaQuery rego.PreparedEvalQuery `yaml:"-"`
}
```

Reply:
It's just easier to have everything in one file and spares us a lot of code duplication.

```rego
package limes

violation[{ "msg": msg, "service": "shared", "resource": "things" }] {
  input.targetdomainreport....
}

violation[{ "msg": msg, "service": "shared", "resource": "things" }] {
  input.targetprojectreport.....
}
```

Reply:
I'm willing to accept a bit of code duplication for increased clarity. When someone looks at the policy in our config two years down the line, I don't want there to be any unnecessary confusion between what is a domain-level policy and what is a project-level policy. Both look so similar (both in structure and in names like `targetdomainreport` vs. `targetprojectreport`) that they will be very likely confused.